source/includes/fundamentals/code-snippets/IndexPage.java

```diff
@@ -48,109 +49,136 @@ public static void main(String[] args) {
-        // error
         try {
             String resultCreateIndex = collection.createIndex(Indexes.text("plot"));
             System.out.println(String.format("Index created: %s", resultCreateIndex));
+        
+        // Prints a message if a text index already exists with a different configuration 
         } catch (MongoCommandException e) {
             if (e.getErrorCodeName().equals("IndexOptionsConflict"))
                 System.out.println("there's an existing text index with different options");
         }
         // end text index
 
+        // Retrieves matching documents directly from the text index, applying a projection
         // begin text query
         Bson filter = text("java coffee shop");
         Bson projection = fields(include("fullplot"), excludeId());
         FindIterable<Document> cursor = collection.find(filter).projection(projection);
         // end text query
+
+        // Prints the results of the find operation
         cursor.forEach(doc -> System.out.println(doc));
     }
 
     private void geoSpatialIndex() {
         System.out.println("geospatial index");
         collection = database.getCollection("theaters");
+
+        // Creates a geospatial index on the "location.geo" field
         // begin geospatial index
-        // if an existing geo index exists, this will error
         try {
             String resultCreateIndex = collection.createIndex(Indexes.geo2dsphere("location.geo"));
             System.out.println(String.format("Index created: %s", resultCreateIndex));
+        
+        // Prints a message if a geospatial index already exists with a different configuration 
         } catch (MongoCommandException e) {
             if (e.getErrorCodeName().equals("IndexOptionsConflict"))
-                System.out.println("there's an existing text index with different options");
+                System.out.println("there's an existing geospatial index with different options");
         }
         // end geospatial index
 
         // begin geospatial query
-        // MongoDB Headquarters in NY, NY.
+        // Stores the coordinates of the NY MongoDB headquarters
         Point refPoint = new Point(new Position(-73.98456, 40.7612));
+
+        // Retrieves documents that represent locations up to 1000 meters from the specified point directly from the geospatial index
+        // Creates a filter to match a document 
         Bson filter = near("location.geo", refPoint, 1000.0, 0.0);
         FindIterable<Document> cursor = collection.find(filter);
         // end geospatial query
+
+        // Prints the results of the find operation
         cursor.forEach(doc -> System.out.println(doc));
     }
 
     private void uniqueIndex() {
         System.out.println("unique index");
         collection = database.getCollection("theaters");
 
+        // Creates a unique index on the "theaterID" field in descending order
         // begin unique index
```

Comment:
Delete this comment
```suggestion
        // begin unique index
```

Reply:
I think this comment should be kept since it is used for the literalinclude directive [here](https://github.com/mongodb/docs-java/blob/master/source/fundamentals/indexes.txt#L440).

In the [Code Comment Style Guide](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit#heading=h.xv6glj2lnzjh), it's identified as a "Marker" comment type.